docs/administration/index.md

```diff
@@ -0,0 +1,14 @@
+# Administration
+
+**Deploying REANA on your premises?**
+
+- [Deploying locally](deployment/deploying-locally) on your laptop using _Kubernetes in Docker_
+- [Deploying at scale](deployment/deploying-at-scale) on Kubernetes clusters using Helm
+
+**Configuring REANA platform?**
+
+- [Configuring user acess](configuration/configuring-access)
+
+**Managing users in REANA?**
```

Comment:
"Managing running REANA platform?" in order to have an open title for other forthcoming things, managing disk space and whatnot.  

P.S. I thought about adding "running" in order to better indicate pre-installation post-installation sections; however perhaps without it the menu would look nicer and more consistent: 

- Deploying REANA platform?
- Configuring REANA platform?
- Managing REANA platform?

Reply:
+1 for consistent option. I've made changes accordingly, here it's how it looks like:
![Screenshot from 2021-01-07 09-50-51](https://user-images.githubusercontent.com/4990025/103871979-0908f880-50ce-11eb-81b6-4d772d9aa788.png)